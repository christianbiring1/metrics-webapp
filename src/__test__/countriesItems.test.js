import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import CountryItem from '../components/countriesItem';
import store from '../redux/configureStore';

it('matches Countries Item', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <BrowserRouter>
        <CountryItem />
      </BrowserRouter>
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
