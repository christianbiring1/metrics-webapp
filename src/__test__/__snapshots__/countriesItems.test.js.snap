// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Countries Item 1`] = `
<section>
  <header
    className="app-header"
  >
    <div
      className="left-header"
    >
      <img
        alt="arrow"
        className="arrow img"
        src="left-arrow.png"
      />
      <h2
        className="year"
      >
        2022
      </h2>
    </div>
    <h3
      className="most-views"
    >
      most views
    </h3>
    <div
      className="img-right"
    >
      <img
        alt=""
        className="img"
        src="mic.png"
      />
      <img
        alt=""
        className="img"
        src="settings.png"
      />
    </div>
  </header>
  <div
    className="home-top"
  >
    <img
      alt="map"
      className="map"
      src="world.svg"
    />
    <div
      className="total"
    >
      <h2>
        WORLDWIDE
      </h2>
      <span>
        0
         
        views
      </span>
    </div>
  </div>
  <p
    className="statbycountry"
  >
    <span>
      STATS BY COUNTRY
    </span>
  </p>
  <div
    className="countries-container"
  />
</section>
`;
